fusion-plugin-web-app-manifest/src/server.js

```diff
@@ -0,0 +1,44 @@
+/** Copyright (c) 2019 Uber Technologies, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ */
+import {createPlugin, html} from 'fusion-core';
+import type {FusionPlugin} from 'fusion-core';
+import {WebAppManifestToken} from './tokens';
+import type {DepsType} from './types';
+
+const plugin =
+  __NODE__ &&
+  createPlugin({
+    deps: {
+      manifest: WebAppManifestToken,
+    },
+    provides({manifest}) {
```

Comment:
If the provides is just returning an exact copy of the value registered on `WebAppManifestToken`, we actually don't need the `provides` at all. If someone depends on the manifest, they can depend on the `WebAppManifestToken` directly.

Then we could just change the `middleware` to get the manifest from the dependencies, rather than the self provides.

```js
middleware({manifest}) {
  ...
```